pkgs/by-name/ho/homerow/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "Homerow";
+  version = "1.4.1";
+
+  src = fetchzip {
+    extension = "zip";
+    name = "Homerow.app";
+    url = "https://builds.homerow.app/v${finalAttrs.version}/Homerow.zip";
+    hash = "sha256-/Zp62UOvjnj+sN8VTpGC9EZ5cLsjOe/A5ZZkJAx/5Xc=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
```

Comment:
These phases are already no-ops so there's not point setting them manually.  

###### Not required for approval.  Accept or dismiss at your own discretion. (See also https://github.com/NixOS/nixpkgs/issues/387072)